modules/dr-hosted-cluster-within-aws-region-delete.adoc

```diff
@@ -21,6 +21,13 @@ Ensure that the `kubeconfig` file of the destination management cluster is place
 
 . Scale the `deployment` and `statefulset` objects by entering these commands:
 +
+[IMPORTANT]
+====
+Do not scale the stateful set if the value of its `spec.persistentVolumeClaimRetentionPolicy.whenScaled` field is set to `Delete`, because this could lead to a loss of data.
```

Comment:
@atiratree should we add since `StatefulSetAutoDeletePVC feature on by default` from 4.14 do not scale the stateful set ? WDYT ?

Reply:
I think mentioning the feature gate is not necessary. If they do not find the field (e.g. in earlier version) then everything is fine and they can skip the warning.

Reply:
This warning is only being applied to 4.14, so they shouldn't even see this warning in earlier versions. I agree that mentioning the feature gate is likely unnecessary here

Reply:
Thank you both !!